tests/manage/mcg/test_s3_with_java_sdk.py

```diff
@@ -0,0 +1,73 @@
+import logging
+import pytest
+
+from ocs_ci.framework.pytest_customization.marks import (
+    bugzilla,
+    polarion_id,
+    skipif_ocs_version,
+    tier2,
+)
+from ocs_ci.ocs.bucket_utils import upload_objects_with_javasdk
+
+logger = logging.getLogger(__name__)
+
+
+@skipif_ocs_version("<4.9")
+@tier2
+class TestS3WithJavaSDK:
+    @bugzilla("2064304")
+    @polarion_id("OCS-3964")
```

Comment:
you are using parametrize with 2 argvalues so it should be treated as 2 tests, please create separate tests in polarion